content/docs/next-release-v2/frontend-ui.md

```diff
@@ -146,6 +151,13 @@ For traces, the supported template fields are: `duration`, `endTime`, `startTime
 
 Further, the trace template fields are available for substitution in process/logs/tags type when the trace template fields are prefixed with `trace.`. For example: `trace.traceID`, `trace.startTime`.
 
+#### Formatting
+
+In addition to interpolating fields into links, formatting functions can be used. The syntax is `#{field | function}` (eg `#{endTime | epoch_micros_to_date_iso}'`). The available formatting functions and a description of their behavior are:
+| Function | Description | Example |
+| -------- | ----------- | ------- |
+| `epoch_micros_to_date_iso` | Format a date in microseconds since epoch to an ISO date time | `#{endTime \| epoch_micros_to_date_iso}` |
```

Comment:
I can imagine additions to this list. A few that are on my mind to being
* left padding traceId with "0"s, for example in jaeger I have a trace ID "42c7668cd8d887da" which shows up in my logs as "000000000000000042c7668cd8d887da" with the OTel pino instrumentation
* Allowing time offsets to be passed to `epoch_micros_to_date_iso` to allow for time skew (eg something like `#{endTime | epoch_micros_to_date_iso 10}` which would add 10 microseconds to the end of the time before formatting it, syntax TBD of course)